src/app/Mappings/MappingsPage.tsx

```diff
@@ -8,18 +8,13 @@ import {
   Tab,
   TabTitleText,
 } from '@patternfly/react-core';
-import { Mapping, MappingType } from '@app/queries/types';
+import { Mapping, MappingType, MapType } from '@app/queries/types';
```

Comment:
I just noticed that you had introduced MapType in the last PR. I don't think it's necessary, right? `MappingType` is the same enum. I've been considering switching it to a string union (`'Network' | 'Storage'`) anyway which would be even simpler. Same for a lot of the other enums.

Reply:
Right, I was following suit from when tabKey was a number. I simplified it to use a string and removed the redundant enum. Good call.